quickstarts/host-inventory-quickstart.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: host-inventory
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    capability.openshift.io/name: Console
+spec:
+  displayName: Set up your on-premise host inventory
+  durationMinutes: 10
+  description: Configure your host inventory and create a cluster.
+  prerequisites: [A storage operator is required (e.g. OpenShift Data Foundation) in order for the assisted service to run correctly.]
```

Comment:
@jhadvig @spadgett Is there a way to link "OpenShift Data Foundation" to the quick start "quickstart?quickstart=odf-install-tour"?